src/library/scala/collection/mutable/OpenHashMap.scala

```diff
@@ -92,8 +95,18 @@ extends AbstractMap[Key, Value]
     deleted = 0
   }
 
+  /** Return the index of the first slot in the hash table (in probe order)
+    * that either is empty, or is or was last occupied by the given key.
+    */
   private[this] def findIndex(key: Key) : Int = findIndex(key, hashOf(key))
 
+  /** Return the index of the first slot in the hash table (in probe order)
+    * that either is empty, or is or was last occupied by the given key.
```

Comment:
Judging from the `while` loop, the returned index will either be such that `table(index) == null` or `table(index).key == key`. To avoid confusion, I would write:

     *  that either is empty, or is occupied by an entry with the given key.

(the "was" part requires context to understand, so I suggest to remove it)